packages/core/src/utils/sourceMaps.ts

```diff
@@ -0,0 +1,5 @@
+import { globalObject } from '../globalObject';
+
+export function getBundleId(appName: string) {
```

Comment:
TS cant't infer teh correct return tyoe and falls back to `any`.
Let's add a return type. `string | undefined`